src/app/public/modules/radio/radio.component.ts

```diff
@@ -91,15 +101,42 @@ export class SkyRadioComponent implements OnDestroy, ControlValueAccessor {
     return this._disabled;
   }
 
+/**
+ * Specifies an ID for the radio button.
+ * @default a unique, auto-incrementing integer. For example: `sky-radio-1`
+ */
   @Input()
   public id = `sky-radio-${++nextUniqueId}`;
 
+/**
+ * Defines an ARIA label for the radio button to support
+ * [accessibility](https://developer.blackbaud.com/skyux-forms/docs/radio#accessibility) when the
+ * radio button does not include a visible label. This property must be set if you are using an
+ * icon radio button. If the radio button includes a visible label on the screen, use the
+ * `labelledBy` property instead.
+ */
   @Input()
   public label: string;
 
+/**
+ * Sets the radio button's `aria-labelledby` attribute to support
+ * [accessibility](https://developer.blackbaud.com/skyux-forms/docs/radio#accessibility). The value
+ * should be the HTML element ID (without the leading #) of the element that labels the radio
```

Comment:
I'm pretty sure `#` should be a code snippet. I checked the existing docs, and it's a code snippet in modal and flyout but not checkout and radio button. I think that's an oversight on the latter 2 components and that we should make it a code snippet here, but it can't hurt to check with Alex or another dev.

Reply:
Looks like this wasn't addressed. Not a huge deal, but just pointing it out so that we can either address it or resolve this comment thread as "will not address." Do you want me to follow up with Alex to determine whether `#` should be a code snippet since we have it both ways in the existing docs?

Reply:
Made `#` a code snippet